jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -316,11 +316,11 @@ optionalt<symbolt> java_bytecode_convert_methodt::get_lambda_method_symbol(
   const java_class_typet::java_lambda_method_handlest &lambda_method_handles,
   const size_t index)
 {
-  const symbol_exprt &lambda_method_handle = lambda_method_handles.at(index);
+  const irept &lambda_method_handle = lambda_method_handles.at(index);
```

Comment:
:question: Why not just an `irep_idt`

Reply:
That's because we want to store this in an `irept` (or a class derived from it), and the only places we can do so for an `irept` are 1) its `id()` - would only permit a single `irep_idt`, 2) its `sub` (is done here), or 3) its `named_sub` (would ensure and require that entries are unique).